sdk/eventhub/event-hubs/tsconfig.json

```diff
@@ -35,6 +35,6 @@
     "resolveJsonModule": true
   },
   "compileOnSave": true,
-  "exclude": ["node_modules", "typings/**", "./samples/**/*.ts"],
+  "exclude": ["node_modules", "types/**", "./samples/**/*.ts"],
```

Comment:
There must be another `tsconfig.json` file under samples folder which `exclude`s `typings`.

Reply:
Good catch, updated!